include/fmt/format-inl.h

```diff
@@ -145,6 +145,61 @@ template <> FMT_FUNC int count_digits<4>(detail::fallback_uintptr n) {
   return i >= 0 ? i * char_digits + count_digits<4, unsigned>(n.value[i]) : 1;
 }
 
+template <typename T = void> struct basic_inl_data {
```

Comment:
I suggest renaming `basic_inl_data` to `basic_impl_data` since this is implementation-specific data while the `-inl` suffix is only relevant to headers.

Reply:
Done